app/dashboard/views.py

```diff
@@ -4311,6 +4311,9 @@ def new_hackathon_bounty(request, hackathon=''):
     except HackathonEvent.DoesNotExist:
         return redirect(reverse('get_hackathons'))
 
+    if hackathon_event.is_expired():
+        return redirect(f"/hackathon/{hackathon_event.slug}/onboard")
```

Comment:
let's use the url name like L4312

`return redirect(reverse('hackathon_onboard'))`

Reply:
updated 🚀